### PR TITLE
Support FT.CONFIG command

### PR DESCRIPTION
**Pull request by dengliming**
As title.

### CODE REVIEW

File: test/test.py [hunk 1]
```diff
@@ -658,6 +658,13 @@ def testGet(self):
         self.assertEqual([['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc2'))
         self.assertEqual([['f1', 'some valid content dd1', 'f2', 'this is sample text ff1'], ['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc1', 'doc2'))
 
+    def testConfig(self):
+        client = self.getCleanClient('idx')
+        self.assertTrue(client.set_config('TIMEOUT', '100'))
```
Comment (gkorland) [inline test/test.py:663]: can you please add negative tests too?

Comment (dengliming) [reply test/test.py:663]: Thanks for reviewing, It was done.

File: redisearch/client.py [hunk 2]
```diff
@@ -507,3 +508,30 @@ def dict_dump(self, name):
         cmd = [self.DICT_DUMP_CMD, name]
         raw = self.redis.execute_command(*cmd)
         return raw
+
+    def set_config(self, option, value):
```
Comment (itamarhaber) [inline redisearch/client.py:512]:
```suggestion
    def config_set(self, option, value):
```

File: redisearch/client.py [hunk 3]
```diff
@@ -507,3 +508,30 @@ def dict_dump(self, name):
...
+        ### Parameters
+
+        - **option**: the name of the configuration option.
+        - **value**: a value for the configuration option.
+        """
+        cmd = [self.CONFIG_CMD, 'SET', option, value]
+        raw = self.redis.execute_command(*cmd)
+        return raw == 'OK'
+
+    def get_config(self, option):
```
Comment (itamarhaber) [inline redisearch/client.py:524]:
```suggestion
    def config_get(self, option):
```

**Comment by itamarhaber** [comment]
Looks good but for consistency, I'd rename the methods.

**Comment by dengliming** [comment]
@itamarhaber Thanks for reviewing. It was done.

File: test/test.py [hunk 4]
```diff
@@ -658,6 +658,14 @@ def testGet(self):
         self.assertEqual([['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc2'))
         self.assertEqual([['f1', 'some valid content dd1', 'f2', 'this is sample text ff1'], ['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc1', 'doc2'))
 
+    def testConfig(self):
+        client = self.getCleanClient('idx')
```
Comment (ashtul) [inline test/test.py:662]: Thanks again for your contributions.
Can you please add a test for TIMEOUT before setting it and a test for a text-based config option such as timeoutpolicy or gcpolicy.

**Pull request merged**